### ISSUE TITLE
NbTabset throws ExpressionChangedAfterItHasBeenCheckedError when NbTabs are generated through *ngFor

### ISSUE THREAD

**Issue by cj-wang** [akveo/nebular#727]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description

**Current behavior:**
NbTabset throws below error when NbTabs are generated through *ngFor:
ERROR Error: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'content-active: false'. Current value: 'content-active: true'.

**Expected behavior:**
Don't throw this error.

**Related code:**
```
    <nb-tabset >
      <nb-tab [tabTitle]="tab" *ngFor="let tab of ['tab1', 'tab2']">
        <p>{{tab}}</p>
      </nb-tab>
    </nb-tabset>
```
As a workaround, this can be fixed by marking the first tab as active explicitly:
```
<nb-tab [tabTitle]="tab" [active]="tabIndex == 0" *ngFor="let tab of ['tab1', 'tab2']; index as tabIndex">
```

**Comment by nnixaa** [comment]
Hi @cj-wang, thanks for reporting!

**Comment by lexzhukov** [comment]
Hello @cj-wang, I can't reproduce this bug, but it will be great if you will create a reproducible example on stackblitz.

**Comment by cj-wang** [comment]
Thanks @lexzhukov. Turns out this has been fixed since 2.0.0-rc.10.  Should have checked the new versions first as my project started from ngx-admin starter-kit branch, still with rc9,

**Comment by cj-wang** [comment]
Closing this issue.
Thanks @nnixaa , @lexzhukov.
